inject.sh

```diff
@@ -1,135 +1,140 @@
 #!/bin/bash
+function replace_or_insert() {
+    # Voodoo magic: https://superuser.com/a/976712
+    grep -q "^${1}=" /conf/.env && sed "s/^${1}=.*/${1}=${2}/" -i /conf/.env || sed "$ a\\${1}=${2}" -i /conf/.env
```

Comment:
```suggestion
    grep -q "^${1}=" /conf/.env && sed -i "s|^${1}=.*|${1}=${2}|" /conf/.env || sed "$ a\\${1}=${2}" -i /conf/.env
```
That should fix the `TIMEZONE` issue

Reply:
I will test that.

Reply:
Tested with that and reverting the TIMEZONE change. I'll push to the branch then let you take a look and merge when you're happy.